font-family

There is an error occurring when building statistics. the issue is importing font-family in ```theme.less```. [See here](https://github.com/mb-wali/invenio-theme-tugraz/blob/47fd19c5719d84415942a3ced234031b18c827b3/invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/theme.less#L12).

here is the error log.
```
@import url("https://fonts.googleapis.com/css?family=Source+Sans+Pro&display=swap");
     ^
     The "url" argument must be of type string. Received an instance of Object
           Error in /opt/invenio/var/instance/assets/less/invenio_theme_tugraz/theme.less (line 12, column 0)
 Child mini-css-extract-plugin node_modules/css-loader/index.js??ref--8-1!node_modules/less-loader/dist/cjs.js??ref--8-2!node_modules/semantic-ui-less/semantic.less:
     Entrypoint mini-css-extract-plugin = *
        18 modules
 npm ERR! code ELIFECYCLE
 npm ERR! errno 2
 npm ERR! invenio-assets@1.0.0 build: `NODE_PRESERVE_SYMLINKS=1 NODE_ENV=production webpack --config ./build/webpack.config.js`
 npm ERR! Exit status 2
 npm ERR! 
 npm ERR! Failed at the invenio-assets@1.0.0 build script.
 npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
 npm ERR! A complete log of this run can be found in:
 npm ERR!     /root/.npm/_logs/2020-07-14T11_38_55_513Z-debug.log
 Traceback (most recent call last):
   File "/opt/rh/rh-python36/root/usr/bin/invenio", line 8, in <module>
     sys.exit(cli())
   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/click/core.py", line 829, in __call__
     return self.main(*args, **kwargs)
   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/flask/cli.py", line 586, in main
     return super(FlaskGroup, self).main(*args, **kwargs)
   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/click/core.py", line 782, in main
     rv = self.invoke(ctx)
   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/click/core.py", line 1259, in invoke
     return _process_result(sub_ctx.command.invoke(sub_ctx))
   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/click/core.py", line 1289, in invoke
     rv.append(sub_ctx.command.invoke(sub_ctx))
   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
     return ctx.invoke(self.callback, **ctx.params)
   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/click/core.py", line 610, in invoke
     return callback(*args, **kwargs)
   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/click/decorators.py", line 21, in new_func
     return f(get_current_context(), *args, **kwargs)
   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/flask/cli.py", line 426, in decorator
     return __ctx.invoke(f, *args, **kwargs)
   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/click/core.py", line 610, in invoke
     return callback(*args, **kwargs)
   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/flask_webpackext/cli.py", line 64, in build
     current_webpack.project.build(*args)
   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/pywebpack/helpers.py", line 48, in inner
     raise RuntimeError("Process exited with code {}".format(exit_code))
 RuntimeError: Process exited with code 2
```

--- a/invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/theme.less
+++ b/invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/theme.less
@@ -9,7 +9,8 @@
 @import "header";
 @import "footer";
 @import "overrides";
-@import url("https://fonts.googleapis.com/css?family=Source+Sans+Pro&display=swap");
+@import (css)
+  url("https://fonts.googleapis.com/css?family=Source+Sans+Pro&display=swap");
 
 /***
 **TODO:

